lib/Page.js

```diff
@@ -79,16 +78,15 @@ class Page extends EventEmitter {
     this._keyboard = new Keyboard(client);
     this._mouse = new Mouse(client, this._keyboard);
     this._touchscreen = new Touchscreen(client, this._keyboard);
-    /** @type {!FrameManager} */
```

Comment:
Can you leave this annotation? I'm pretty it is needed because typescript gets confused when we pass `this` as a parameter to an object in our own constructor.